nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -61,14 +60,14 @@ InflationLayer::InflationLayer()
   inscribed_radius_(0),
   cost_scaling_factor_(0),
   inflate_unknown_(false),
+  inflate_around_unknown_(false),
   cell_inflation_radius_(0),
   cached_cell_inflation_radius_(0),
-  cached_costs_(nullptr),
-  cached_distances_(nullptr),
-  last_min_x_(-std::numeric_limits<float>::max()),
-  last_min_y_(-std::numeric_limits<float>::max()),
-  last_max_x_(std::numeric_limits<float>::max()),
-  last_max_y_(std::numeric_limits<float>::max())
+  cache_length_(0),
+  last_min_x_(std::numeric_limits<double>::lowest()),
```

Comment:
lowest appears to just be -max  ( https://en.cppreference.com/w/cpp/types/numeric_limits). Not sure what issue you were referring to that asked for that to be changed. Can you clarify / link?